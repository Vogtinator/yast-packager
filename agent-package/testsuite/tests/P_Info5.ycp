{
return Execute(.setEnvironment,$["packageinfopath":"tests","language":"de","common.pkd":"common.ori.pkd","dudir":"tests/du.dir","partition":[$["name":"/","used":0,"free":10000],$["name":"bin","used":0,"free":10000],$["name":"etc","used":0,"free":10000],$["name":"sbin","used":0,"free":10000],$["name":"usr","used":0,"free":10000],$["name":"var","used":0,"free":10000]]]);
}
{
    return Read(.selGroups);
}
{
    return Execute(.setEnvironment,$["packageinfopath":"tests","language":"cs_CZ","common.pkd":"common.ori.pkd","dudir":"tests/du.dir","partition":[$["name":"/","used":0,"free":10000],$["name":"bin","used":0,"free":10000],$["name":"etc","used":0,"free":10000],$["name":"sbin","used":0,"free":10000],$["name":"usr","used":0,"free":10000],$["name":"var","used":0,"free":10000]]]);
}
{
    return Read(.selGroups);
}
{
    return Execute(.setEnvironment,$["packageinfopath":"tests","language":"fr_FR","common.pkd":"common.ori.pkd","dudir":"tests/du.dir","partition":[$["name":"/","used":0,"free":10000],$["name":"bin","used":0,"free":10000],$["name":"etc","used":0,"free":10000],$["name":"sbin","used":0,"free":10000],$["name":"usr","used":0,"free":10000],$["name":"var","used":0,"free":10000]]]);
}
{
    return Read(.selGroups);
}
