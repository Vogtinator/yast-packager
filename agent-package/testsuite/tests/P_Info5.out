(true)
([["Devel", " Entwicklung", "addon"], ["Games", " Spiele", "addon"], ["Gnome", " Gnome System", "addon"], ["Kde", " KDE komplett", "addon"], ["Kde-Desktop", " KDE Desktop Environment", "addon"], ["LAMP", " Simple Webserver", "addon"], ["Min", " Minimum system", "baseconf"], ["Minimal", " Minimal System", "baseconf"], ["Minimal+X11", " Minimales graphisches System (ohne KDE) ", "baseconf"], ["Multimed", " Multimedia", "addon"], ["Multimedia", " Multimedia", "addon"], ["Network", " Netzwerk/Server", "addon"], ["Standard", " Standard-System", "baseconf"], ["default", " Standard-System mit Office", "baseconf"], ["xxxx", "  german xxxx", "baseconf"], ["zzz_Alles", " Alle Pakete (sehr zeitaufwendig)", "baseconf"]])
(true)
([["Devel", " V√Ωvoj", "addon"], ["Games", " Hry", "addon"], ["Gnome", " Syst√©m Gnome", "addon"], ["Kde", " V≈°e z KDE", "addon"], ["Kde-Desktop", " KDE Desktop Environment", "addon"], ["LAMP", " Simple Webserver", "addon"], ["Min", " Minimum system", "baseconf"], ["Minimal", " Minim√°ln√≠ syst√©m", "baseconf"], ["Minimal+X11", " Minim√°ln√≠ grafick√Ω syst√©m (bez KDE) ", "baseconf"], ["Multimed", " Multim√©dia", "addon"], ["Multimedia", " Multim√©dia", "addon"], ["Network", " S√≠≈•/Server", "addon"], ["Standard", " Standardn√≠ syst√©m", "baseconf"], ["default", " Syst√©m s kancel√°≈\231sk√Ωm bal√≠kem", "baseconf"], ["xxxx", " xxxx", "baseconf"], ["zzz_Alles", " V≈°echny bal√≠ky (trv√° velmi dlouho)", "baseconf"]])
(true)
([["Devel", " D√©veloppement", "addon"], ["Games", " Jeux", "addon"], ["Gnome", " Syst√®me Gnome", "addon"], ["Kde", " Tout de KDE", "addon"], ["Kde-Desktop", " KDE Desktop Environment", "addon"], ["LAMP", " Simple Webserver", "addon"], ["Min", " Minimum system", "baseconf"], ["Minimal", " Syst√®me minimal", "baseconf"], ["Minimal+X11", " Syst√®me minimal (X11) (sans KDE)", "baseconf"], ["Multimed", " Multim√©dia", "addon"], ["Multimedia", " Multim√©dia", "addon"], ["Network", " R√©seau/Serveur", "addon"], ["Standard", " Syst√®me standard", "baseconf"], ["default", " Syst√®me standard avec Office", "baseconf"], ["xxxx", " xxxx", "baseconf"], ["zzz_Alles", " Tous les paquetages (prend du temps!)u", "baseconf"]])
